kura/test/org.eclipse.kura.nm.test/src/test/java/org/eclipse/kura/nm/configuration/NMSettingsConverterTest.java

```diff
@@ -452,6 +454,34 @@ public void buildIpv6SettingsShouldThrowIllegalArgumentExceptionWhenGivenExpecte
 
         thenIllegalArgumentExceptionThrown();
     }
+    
+    @Test
+    public void buildIpv6SettingsShouldHaveMtuWhenSupported() {
+    	givenMapWith("net.interface.wlan0.config.ip6.address.method", "netIPv6MethodAuto");
+        givenMapWith("net.interface.wlan0.config.ip6.status", "netIPv6StatusEnabledWAN");
+        givenMapWith("net.interface.wlan0.config.ip6.mtu", 2345);
+        givenNetworkPropsCreatedWithTheMap(this.internetNetworkPropertiesInstanciationMap);
+        givenNetworkManagerVersion("1.40.18");
+
+        whenBuildIpv6SettingsIsRunWith(this.networkProperties, "wlan0");
```

Comment:
Can you please update the `whenBuildIpv6SettingsIsRunWith` signature so that it accepts the version as an argument? This is so that we're consistent throughout the codebase.

```suggestion
        givenNetworkManagerVersion("1.40.18");

        whenBuildIpv6SettingsIsRunWith(this.networkProperties, "wlan0", this.nmVersion);
```